react/TextField/TextField.js

```diff
@@ -72,20 +74,22 @@ export default class TextField extends Component {
     this.handleMouseDownOnClear = this.handleMouseDownOnClear.bind(this);
   }
 
+  storeContainerReference = textField => {
+    if (textField !== null) {
+      this.container = textField;
+    }
+  }
+
   storeInputReference(input) {
     if (input !== null) {
       this.input = input;
     }
   }
 
   handleMouseDownOnClear(event) {
-    const { onClear } = this.props;
-
-    if (typeof onClear === 'function') {
```

Comment:
It's like if clear function is not passed, when you're trying to clear the field don't focus it...can't imagine the case it's needed